imap_processing/hit/packet_definitions/P_HIT_HSKP.xml

```diff
@@ -63,7 +71,7 @@
 				<xtce:LongDescription>CCSDS Packet Length (number of bytes after Packet length minus 1)</xtce:LongDescription>
 			</xtce:Parameter>
 			<xtce:Parameter name="SHCOARSE" parameterTypeRef="UINT32">
```

Comment:
Yeah, I did remember to change the updated sheet to SHOCOARSE for housekeeping since this is the one I was working on for this PR. Going to remove all non-hk xtce  for this PR